Dockerfile

```diff
@@ -34,12 +36,26 @@ COPY tdm-be /usr/src/app
 # Build the node app
 RUN npm run build
 
+COPY /usr/src/app/dist /usr/src/app
+
 # Copy built React app from the previous stage
 COPY --from=react-build /app/.next /usr/src/app/public/_next
 
 RUN mkdir /usr/src/app/public/downloads
+
+# Add ezmaster config file
+RUN echo '{ \
+    "httpPort": 3000, \
+    "configPath": "/usr/src/app/config.json", \
+    "dataPath": "/usr/src/app/public/downloads", \
+    }' > /etc/ezmaster.json  && \
+    sed -i -e "s/daemon:x:2:2/daemon:x:1:1/" /etc/passwd && \
+    sed -i -e "s/daemon:x:2:/daemon:x:1:/" /etc/group && \
+    sed -i -e "s/bin:x:1:1/bin:x:2:2/" /etc/passwd && \
+    sed -i -e "s/bin:x:1:/bin:x:2:/" /etc/group
+
 # Expose the port your Express server will run on
 EXPOSE 3000
 
 # Start the Express app
-CMD ["npm", "start"]
\ No newline at end of file
+CMD ["npm", "start:prod"]
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏start appli as user `daemon`

lodex uses `su -exec daemon:daemon` to launch appli, so that the `dataPath` could be written by webdav, if I remember correctly.

Reply:
See https://github.com/Inist-CNRS/lodex/blob/master/docker-entrypoint.sh